test/kldivergences.jl

```diff
@@ -0,0 +1,78 @@
+@testset "KL divergences" begin
```

Comment:
I just noticed that there are already some tests for `kldivergence` in `test/functionals.jl`. Can you move the tests here to this file?